OracleCoherence/README.md

```diff
@@ -1,115 +1,193 @@
+<!--
+    Copyright (c) 2015, 2020, Oracle and/or its affiliates.
+    Licensed under the Universal Permissive License v 1.0 as shown at
+    http://oss.oracle.com/licenses/upl.
+-->
 Oracle Coherence Docker Image
 ===============
-This section is about using [Oracle Coherence](http://www.oracle.com/technetwork/middleware/coherence/overview/index.html) in Docker. The purpose of the Docker images described here is to facilitate the setup of development and integration testing environments for developers. This project includes example dockerfiles and documentation for Coherence 12.2.1 based on Oracle Linux and Oracle JDK 8.
+This section is about using [Oracle Coherence](http://www.oracle.com/technetwork/middleware/coherence/overview/index.html)
+in Docker. The purpose of the Docker images described here is to facilitate the setup of development
+and integration testing environments for developers. This project includes example Dockerfiles and
+documentation for Coherence based on Oracle Linux and Oracle JDK 8.
 
-The certification of Coherence on Docker does not require the use of any file presented in this repository. Customers and users are welcome to use them as starters, and customize/tweak, or create from scratch new scripts and Dockerfiles.
+The certification of Coherence on Docker does not require the use of any file presented in this
+repository. Customers and users are welcome to use them as starters, and customize/tweak, or create
+from scratch new scripts and Dockerfiles.
 
 ## Oracle Linux Base Image
 For more information and documentation, read the [Docker Images from Oracle Linux](https://registry.hub.docker.com/_/oraclelinux/) page.
 
 ### Standalone Distribution
-For more information on the Coherence Standalone Distribution, visit [Coherence 12.2.1 Documentation](http://docs.oracle.com/middleware/12214/coherence/index.html).
+For more information on the Coherence Standalone Distribution, visit the
+[Coherence Documentation](https://docs.oracle.com/en/middleware/standalone/coherence/index.html).
 
 ## Building Oracle JDK (Server JRE) base image
-Before you can build these Oracle Coherence images you must have built the required Oracle Java 8 base image (see [Oracle Java images](../OracleJava/))
+Before you can build these Oracle Coherence images you must have built the required Oracle Java 8
```

Comment:
There appears to be an option to build using Graal as well. Is the Graal image required to be build manually or do you pull that automatically? Either way, it should be documented and an option to build the Graal image should be provided so that users who have no access to images on the Docker Hub can still build Coherence using Graal.

Reply:
I updated the base image instructions to reference the GraalVM instructions as well.